specification/monitor/resource-manager/microsoft.insights/stable/2018-04-16/scheduledQueryRule_API.json

```diff
@@ -399,7 +400,6 @@
         }
       },
       "required": [
-        "query",
```

Comment:
Why this property is not required anymore?

Reply:
This property is optional as the LogToMetric action of the api does not need this property.

Reply:
@jagpreetstamber This is stable API and it is publicly available since April 2018. It means your customers may already have code which is using your SDKs (C#, TypeScript, Go, etc). This specific change may introduce breaking changes to their code. I would suggest to introduce the changes in the next API version unless you are ok to have breaking changes for your customers.

Reply:
@sergey-shandar We have discussed this internally and agreed on making this change.